tests/foreman/cli/test_repository.py

```diff
@@ -2566,18 +2566,17 @@ def test_positive_symlinks_sync(self):
         :CaseAutomation: automated
         """
         # Setting Creating Local Directory using Pulp Manifest and
```

Comment:
nitpick, should we update the comment if we no longer use pulp manifest?

Reply:
sure will do that